components/RecipeForm/RecipeForm.tsx

```diff
@@ -50,120 +55,224 @@ const emptyRecipe: Recipe = {
-      setFormData((currData) => ({
-        ...currData,
+      setRecipeData((existingRecipeData) => ({
+        ...existingRecipeData,
         [name]: parseInt(value),
       }));
     }
   };
 
-  const handleInstructionsChange = (
-    event: ChangeEvent<HTMLTextAreaElement>
-  ) => {
-    const { value } = event.target;
-
-    setFormData((currData) => {
-      const newInstructions = [...currData.instructions];
-      newInstructions[0] = { ...newInstructions[0], description: value };
-      return {
-        ...currData,
-        instructions: newInstructions,
-      };
+  /**
+   * Handles changes to individual ingredient fields
+   */
+  const handleIngredientChange = (newIngredient: Ingredient) => {
+    setRecipeData((existingRecipeData) => {
+      const newIngredients = existingRecipeData.ingredients.map((ingredient) =>
+        ingredient.id === newIngredient.id ? newIngredient : ingredient
+      );
+      return { ...existingRecipeData, ingredients: newIngredients };
     });
   };
 
+  /**
+   * Adds a new empty ingredient to the ingredients array
+   */
   const handleAddIngredient = () => {
-    setFormData((currData) => {
+    setRecipeData((existingRecipeData) => {
       const newIngredients = [
-        ...currData.ingredients,
+        ...existingRecipeData.ingredients,
         {
           id: crypto.randomUUID(),
           quantity: 0,
           unit: '',
           name: '',
         },
       ];
-      return { ...currData, ingredients: newIngredients };
+      return { ...existingRecipeData, ingredients: newIngredients };
     });
   };
 
-  const handleIngredientChange = (newIngredient: Ingredient) => {
-    setFormData((currData) => {
-      const newIngredients = currData.ingredients.map((ingredient) =>
-        ingredient.id === newIngredient.id ? newIngredient : ingredient
-      );
-      return { ...currData, ingredients: newIngredients };
+  /**
+   * Handles changes to the recipe instructions textarea
+   */
+  const handleInstructionsChange = (
+    event: ChangeEvent<HTMLTextAreaElement>
+  ) => {
+    const { value } = event.target;
+
+    setRecipeData((existingRecipeData) => {
+      const newInstructions = [...existingRecipeData.instructions];
+      newInstructions[0] = { ...newInstructions[0], description: value };
+      return {
+        ...existingRecipeData,
+        instructions: newInstructions,
+      };
     });
   };
 
+  /**
+   * Handles changes to the preparation time inputs
+   */
   const handlePrepTimeChange = (newPrepTime: number) => {
-    setFormData((currData) => ({ ...currData, prepTime: newPrepTime }));
+    setRecipeData((existingRecipeData) => ({
+      ...existingRecipeData,
+      prepTime: newPrepTime,
+    }));
   };
 
+  /**
+   * Handles changes to the cooking time inputs
+   */
   const handleCookingTimeChange = (newCookingTime: number) => {
-    setFormData((currData) => ({ ...currData, cookingTime: newCookingTime }));
+    setRecipeData((existingRecipeData) => ({
```

Comment:
I think this is some kind of personal pref. but I dont like that, everywhere "currData" was changed to "existingRecipeData" -> Because with "currData" or "currentData" -> It is clear, that the currentData of the formObject // the recipe is meant.

Reply:
Thanks! I'll adjust the naming, so that we standardize on a naming convention 👍